tests/util/generateBatchPlateData.ts

```diff
@@ -0,0 +1,114 @@
+import * as fs from 'fs'
+import completeHGVTechRecord from '../resources/techRecordCompleteHGVPlate.json';
+import completeTRLTechRecord from '../resources/techRecordsTrlPost.json';
+import existingSeed from '../resources/technical-records-v3.json';
+import { TechRecordGETHGV, TechRecordGETTRL } from '@dvsa/cvs-type-definitions/types/v3/tech-record/tech-record-verb-vehicle-type';
+
+function padZeroes(number: string, size: number) {
+    number = number.toString();
+    while (number.length < size) number = "0" + number;
+    return number;
+}
+
+let records = []
+let triggerData = []
+console.log("starting")
+//complete HGVs
+for(var i = 1; i<=40000; i++)
```

Comment:
upon running the script, the first couple of records contain the following details

>   "systemNumber": "XYZEP5JYOMM00020",
>     "createdTimestamp": "2019-06-22T10:26:55.903Z",
Doing this is going to cause them to be rejected when trying to put them into any dynamo db due to the overlap between the keys on the table.

Reply:
My understanding is that our current seed process only uses the `technical-records-v3.json` file. This script appends to that with the intention of being used through the exisiting seed process. If our seed pulls together multiple files, that should quite quickly solve the issues we were seeing with the size of the JSON.